specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -920,6 +1549,154 @@
+          "type": "integer",
+          "format": "int32"
+        },
+        "maxNumberOfJobs": {
+          "description": "Max Number of Jobs for Router Job",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "AcsRouterQueueDetails": {
+      "description": "Router Queue Details",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Router Queue Id",
+          "type": "string"
+        },
+        "name": {
+          "description": "Router Queue Name",
+          "type": "string"
+        },
+        "labels": {
+          "description": "Router Queue Labels",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AcsRouterCommunicationError": {
+      "description": "Router Communication Error",
+      "properties": {
+        "code": {
+          "description": "Router Communication Error Code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Router Communication Error Message",
+          "type": "string"
+        },
+        "target": {
+          "description": "Router Communication Error Target",
+          "type": "string"
+        },
+        "innerError": {
+          "description": "Router Communication Inner Error",
+          "$ref": "#/definitions/AcsRouterCommunicationError"
+        },
+        "details": {
+          "description": "List of Router Communication Errors",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterCommunicationError"
+          }
+        }
+      }
+    },
+    "AcsRouterWorkerSelector": {
+      "description": "Router Job Worker Selector",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "Router Job Worker Selector Key",
+          "type": "string"
+        },
+        "labelOperator": {
+          "description": "Router Job Worker Selector Label Operator",
+          "type": "string",
+          "enum": [
+            {
+              "operation": "=",
+              "name": "Equal"
+            },
+            {
+              "operation": "!=",
+              "name": "NotEqual"
+            },
+            {
+              "operation": ">",
+              "name": "Greater"
+            },
+            {
+              "operation": "<",
+              "name": "Less"
+            },
+            {
+              "operation": ">=",
+              "name": "GreaterThanOrEqual"
+            },
+            {
+              "operation": "<=",
+              "name": "LessThanOrEqual"
+            }
+          ],
+          "x-ms-enum": {
+            "name": "acsRouterLabelOperator",
+            "modelAsString": true
+          }
+        },
+        "labelValue": {
+          "description": "Router Job Worker Selector Value",
+          "type": "object"
+        },
+        "ttl": {
```

Comment:
Could this be renamed to timeToLive? If not can you add 
`"x-ms-client-name": "timeToLive"`

Reply:
I will add the "x-ms-client-name": "timeToLive" to the file.